### ISSUE TITLE
scripts/run_clm_sft_with_peft.py  中是在哪里冻结模型参数的呢？代码中没有找到，大佬可以指点一下么？

### ISSUE THREAD

**Issue by huruizhi** [ymcui/Chinese-LLaMA-Alpaca#468]
*提示：将[ ]中填入x，表示打对钩。提问时删除这行。只保留符合的选项。*

### 详细描述问题

scripts/run_clm_sft_with_peft.py  中是在哪里冻结模型参数的呢？代码中没有找到，大佬可以指点一下么？

### 运行截图或日志

*请提供文本log或者运行截图，以便我们更好地了解问题详情。*

### 必查项目（前三项只保留你要问的）

- [ ] **基础模型**：LLaMA / Alpaca / LLaMA-Plus / Alpaca-Plus
- [ ] **运行系统**：Windows / MacOS / Linux 
- [ ] **问题分类**：下载问题 / 模型转换和合并 / 模型训练与精调 / 模型推理问题（🤗 transformers） / 模型量化和部署问题（llama.cpp、text-generation-webui、LlamaChat） / 效果问题 / 其他问题
- [ ] （必选）由于相关依赖频繁更新，请确保按照[Wiki](https://github.com/ymcui/Chinese-LLaMA-Alpaca/wiki)中的相关步骤执行
- [ ] （必选）我已阅读[FAQ章节](https://github.com/ymcui/Chinese-LLaMA-Alpaca/wiki/常见问题)并且已在Issue中对问题进行了搜索，没有找到相似问题和解决方案
- [ ] （必选）第三方插件问题：例如[llama.cpp](https://github.com/ggerganov/llama.cpp)、[text-generation-webui](https://github.com/oobabooga/text-generation-webui)、[LlamaChat](https://github.com/alexrozanski/LlamaChat)等，同时建议到对应的项目中查找解决方案

**Comment by ymcui** [comment]
建议学习一下🤗peft库

**Comment by huruizhi** [comment]
找到了  peft 在调用 add_adapter 函数时，将所有不以`lora_`开头的层 都冻结住了

<img width="1002" alt="image" src="https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/20983257/5ea4da4a-5e91-4952-b5a7-ab69ad2ecc39">


<img width="975" alt="image" src="https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/20983257/56234192-ba64-4b92-a79b-1acb9e93f6f8">

**Comment by sarahyongq** [comment]
> 建议学习一下🤗peft库

run_pt.sh 跑预训练，报错RuntimeError: element 0 of tensors does not require grad and does not have a grad_fn，会有None of the inputs have requires_grad=True. Gradients will be None提示，是因为代码哪里因为冻结了参数导致吗？但是我看run_clm_pt_with_peft.py并没有任何冻结参数训练的设置呢？麻烦看看？
